games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef

```diff
@@ -0,0 +1,39 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+	major = 0,
+	middle = 0,
+	minor = 0,
+	
+	variety = B_APPV_DEVELOPMENT,
+	internal = 0;
```

Comment:
(I'm replying here because GitHub is not showing the "Reply" button on your next comment for some unknown reason to me.)

> $binDir/rocksndiamonds isn't correct here,  it should be:
$appsDir/"Rocks'n'Diamonds"/rocksndiamonds

Yep, I know. That's why I suggested using this in my [review](#discussion_r93825330):
```
	addResourcesToBinaries $portDir/additional-files/rocksndiamonds.rdef \
		$destdir/rocksndiamonds
```

> PS @fbrosson does the iom file need to go into the ADDITIONEL_FILES? as it's not needed for the build ...

Exact. If it's not needed for the build (or install), then we don't need it in ADDITIONEL_FILES.
But wait, where is that file used, if it's not required in the recipe?
Should it be part of the package? If yes then we need to add steps for it (and keep it in ADDITIONEL_FILES, of course.)